File: java-web-performance-webflux/src/main/java/com/binghe/Member.java
package com.binghe;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Table("members")
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@Setter
@ToString
public class Member {

    @Id
    @Column("id")
    private Long id;

    @Column("name")
    private String name;

    @Column("age")
    private int age;

    @Column("description")
    private String description;
}

File: kotlin-grpc-coroutine-spring-boot/grpc-server/src/main/kotlin/com/binghe/member/Member.kt
package com.binghe

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table

@Table("members")
data class Member(
    @Id
    @Column("id")
    val id: Long? = null,

    @Column("name")
    val name: String? = null,

    @Column("age")
    val age: Int? = null,

    @Column("description")
    val description: String? = null
)
